engine/requirements-dev.txt

```diff
@@ -12,3 +12,4 @@ types-PyMySQL==1.0.19.7
 types-python-dateutil==2.8.19.13
 types-requests==2.31.0.1
 httpretty==1.1.4
+pytest-xdist[psutil]==3.5.0
```

Comment:
`psutil` from [the docs](https://pytest-xdist.readthedocs.io/en/stable/#installation):
> To use psutil for detection of the number of CPUs available, install the psutil extra